### ISSUE TITLE
get_python.bat doesn't make permanent changes

### ISSUE THREAD

**Issue by LincePotiguara** [o3de/o3de#1937]
**Describe the bug**
Running `python\get_python` after compilation sets up the python runtime only to the terminal the command was typed in.

**To Reproduce**
1. Follow the required building steps, except step 3
2. Delete or rename python\runtime folder if it exists
3. Run python\get_python.bat
4. Open a new terminal
5. Run Editor.exe through the terminal
6. See error

**Expected behavior**
The configuring process should not be local to the terminal.

**Screenshots**
![error](https://user-images.githubusercontent.com/12141500/124834379-093b6100-df56-11eb-986d-49060e1f9992.PNG)

**Additional context**
It's possible to run Editor.exe normally only after the execution of the batch file, but running Editor.exe from a new terminal or from the Explorer yields the same error.

**Comment by cgalvan** [comment]
Are you able to still reproduce this on the latest development branch? I wasn't able to reproduce this issue. The get_python.bat script doesn't set up any environment variables that would make it only work on that local terminal, but rather it installs to the python/runtime folder, which is accessed by the python/python.cmd.

Is this what you see in your python/runtime folder?

```python-3.7.10-rev1-windows```

**Comment by amznestebanpapp** [comment]
Another common related thing is that if you run the Editor from the "install" location, python needs to be obtained in the install location as well. Assuming default paths:
-  `<source>/python/get_python.bat` is the python that will be used by `<source>/build/windows_vs2019/bin/profile/Editor.exe`
- `<source>/install/python/get_python.bat` is the python that will be used by `<source>/install/bin/Windows/profile/Editor.exe`
You can manually copy the python from one to another, but they are needed at those locations for those.

The difference is that the second case is what is used to pack an installer, so we need a layout of what it would look like in the destination.

**Comment by LincePotiguara** [comment]
@cgalvan yes and yes. I got the same results testing on stabilization/2106 branch.

**Comment by LincePotiguara** [comment]
@amznestebanpapp moving the build folder to same location of the source folder solved it. 
Should this information be on the README?

**Comment by amznestebanpapp** [comment]
Currently the readme is not instructing to do an "install" build. I think there has been some confusion on what that "install" build is and how to use it. 

The "install" build is a layout of what the installer (the wizard) will have. Its meant to be a self-contained engine, so it needs its own python copy. The process of building the install target is not adding python because the installer downloads python after installing. 

I agree this just needs better documentation (which we have not done yet). Tagging it with sig/docs so we think we document the install process better. With better documentation developers should understand what they are building and how to use it.

**Comment by nakhleh** [comment]
I'm trying to get started with o3de and this issue, or something related to it, seems to be a significant barrier to entry to getting started. I am following the tutorials, for example this one: https://o3de.org/docs/welcome-guide/get-started/project-config/creating-projects-using-project-manager/
It clearly says to run the programs like o3de.exe from <install>/bin/Windows/profile, but it doesn't seem to work. I can't get it to find python, whether in the window in which I ran get_python.sh or in another window after having run it. Apps work fine from <repo>/build/bin/profile, but that isn't what the instructions say to do. And, of course, the install dir does have to work, if that's what we're supposed to be able to package up.

**Comment by amznestebanpapp** [comment]
The instructions to run `get_python.bat` are here: https://o3de.org/docs/welcome-guide/setup/setup-from-github/
What instructions did you follow to get the repository and do the first setup? (maybe the instructions are missing there)

`get_python.bat` is considered part of the "setup", not part of "build". Once we have an installer, that will run as part of the installation process.

**Comment by nakhleh** [comment]
In my case, it was my bad. But, not quite as dumb as simply not installing python, so I'll include my mistake here, in case there's a tweak that can be made to reduce the possibility for error for others.
As it turns out, I was using git-bash (as I imagine many folks might do, considering you need git installed to clone the repo, and Windows Command Prompt is objectively terrible ;) ). I was following the (well-written, thanks!) directions at https://o3de.org/docs/welcome-guide/setup/setup-from-github/, but I forgot in step 3 of "Register the Engine" to run o3de.sh instead of o3de.bat. It *looked* like it worked in git-bash, but it didn't actually work. When I went through "Register the Engine" steps using Windows Command Prompt instead of git-bash as my shell, o3de fired right up.

**Comment by amznestebanpapp** [comment]
@nakhleh glad that you were able to figure it out. Environments are a tricky thing, there is such a big amount of shells, configurations, versions, etc out there that guaranteeing a smooth bootstrap is tricky.
I use cmder as my shell and have not had that issue. Maybe something to add to the docs? @o3de/sig-docs
